### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by iamvldmrbrvkv** [mdn/content#35272]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#evaluation_order

### What information was incorrect, unhelpful, or incomplete?

The initializer is evaluated during instance creation, at the start of the constructor (for base classes) or immediately **before** the super() call returns (for derived classes).

### What did you expect to see?

The initializer is evaluated during instance creation, at the start of the constructor (for base classes) or immediately **after** the super() is fully executed (for derived classes).

### Do you have anything more you want to share?

Only when parent class is fully executed  child class initializers starts evaluated and added fields to 'this'.
```js
// The order of evaluation and initialization of fields in parent and child classes
// Definition of ParentClassEvaluation
class ParentClassEvaluation {
  // Step 1: The expression for the initializer of parentField is saved here.
  parentField = "parentIinitializer";
  
  // Step 2: Definition of the parent class constructor
  constructor(firstName) {
    // This is called during the creation of an instance.
    // Evaluation of initializer: This is the moment when, during instance creation, the initializer expression is actually executed and the value is assigned to the field. Thus, at this stage, the initializer expression "parentIinitializer" is assigned to the parentField.
    // this.parentField and this.firstName are available here.
    this.firstName = firstName;
    this.parentField = "newParentField"; // Here parentField is reassigned.
  }
}

// Definition of ChildClassEvaluation (subclass of ParentClassEvaluation)
class ChildClassEvaluation extends ParentClassEvaluation {
  // Step 3: The expression for the initializer of childField is saved here.
  childField = "childdInitializer";
  
  // Step 4: Definition of the child class constructor
  constructor(firstName, secondName) {
    // Step 5: super() is called to initialize the parent class
    super(firstName);
    
    // this.parentField and this.firstName are already initialized and available.
    // After returning from super(), the fields of the current class are initialized.
    // Evaluation of initializer: This is the moment when, during instance creation, the initializer expression is actually executed and the value is assigned to the field. Thus, at this stage, the initializer expression "childdInitializer" is assigned to the childField.
    // this.childField is available here.
    this.secondName = secondName; // secondName is initialized.
    this.childField = "newChildField"; // childField is reassigned here.
  }
}

// Step 6: Creating an instance of ChildClassEvaluation
const instanceFromChildClassEvaluation = new ChildClassEvaluation('Vladimir', 'Borovikov');

// Step 7: Output the instance to see the field values
console.log(
  "instanceFromChildClassEvaluation",
  instanceFromChildClassEvaluation
);

// it's the same to:
function ParentConsructorEvaluation(firstname) {
  this.parentField = "parentIinitializer"
  this.first = firstname
}

function ChildConsructorEvaluation(firstname, secondName) {
  ParentConsructorEvaluation.call(this, firstname) // only after parent constructor finished execution, child field are initialized
  this.childField = "childdInitializer";
  this.second = secondName
}

const instanceFromChildConstructorEvaluation = new ChildConsructorEvaluation('Vladimir', 'Borovikov');

console.log('instanceFromChildConstructorEvaluation', instanceFromChildConstructorEvaluation)
```

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/classes`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/classes/index.md
* Last commit: https://github.com/mdn/content/commit/1b2c87c20466d2a3eec9b3551c269f9aff8f5762
* Document last modified: 2024-07-25T21:53:55.000Z

</details>

**Comment by Josh-Cena** [comment]
Duplicate of https://github.com/mdn/content/issues/35270
